www/js/user_page.js

```diff
@@ -273,12 +275,12 @@ function initUserPage() {
       // Get data from the form
       formData = app.form.convertToData('#user-form');
 
-      // Update custom food data
-      $.auth.user.food_custom = foodCustom;
-
-      // Edit and update formData and the user object
+      // Edit and update formData
       formData = prepareFormData(formData, user);
 
+      // Update custom food data to local user variable
+      user.food_custom = foodCustom;
```

Comment:
I think it's better to move this to `prepareFormData()` where the rest of the local user object is updated